### ISSUE TITLE
Add release notes

### ISSUE THREAD

**Issue by pedrorijo91** [jOOQ/jOOL#334]
Hey,

It would be really useful to understand what's new between each version to have release notes for each release. 

Thanks,
Pedro

**Comment by billoneil** [comment]
You can find all the commits per release here https://github.com/jOOQ/jOOL/milestone/12?closed=1 in the short term.

**Comment by pedrorijo91** [comment]
yes, the other possibility is to use https://github.com/jOOQ/jOOL/compare/version-0.9.12...version-0.9.13

but none of them are very friendly (a short summary is way more convenient)

thanks for the answer :)

**Comment by lukaseder** [comment]
Thank you very much for your suggestion. Yes indeed, we should do this for all of our secondary products and link to the notes from here: https://www.jooq.org/products
